src/box/lua/config/applier/roles.lua

```diff
@@ -260,14 +260,14 @@ local function post_apply(config)
 
     roles_state.last_loaded = loaded
     roles_state.last_roles_ordered = roles_ordered
-    roles_state.last_config = table.deepcopy(config)
+    roles_state.last_config = table.deepcopy(roles_cfg)
```

Comment:
Nit: I would adjust the description at top of the module as well.

Maybe it alsoworth to rename it to `last_roles_cfg` to make it more obvious what is stored here.

Reply:
I was thinking about the renaming too, but at first decided to keep the changes minimal. Now that you mentioned it - renamed)